test/runner/lib/util.py

```diff
@@ -170,8 +170,8 @@ def raw_command(cmd, capture=False, env=None, data=None, cwd=None, explain=False
         encoding = 'utf-8'
         data_bytes = data.encode(encoding) if data else None
         stdout_bytes, stderr_bytes = process.communicate(data_bytes)
-        stdout_text = stdout_bytes.decode(encoding) if stdout_bytes else u''
-        stderr_text = stderr_bytes.decode(encoding) if stderr_bytes else u''
+        stdout_text = stdout_bytes.decode(encoding, 'replace') if stdout_bytes else u''
+        stderr_text = stderr_bytes.decode(encoding, 'replace') if stderr_bytes else u''
```

Comment:
This is used by `ansible-test` to execute all external commands, so I'm not sure this is a good idea. I believe the only encoding issue is with parsing of `git diff` output, as seen from an older CI run for this PR:

https://app.shippable.com/github/ansible/ansible/runs/29803/1/console

I'm thinking that `replace` should only be used for `git diff` output, since it's not guaranteed to be `utf-8`.

@abadger Does that seem reasonable to you?